### ISSUE TITLE
Can we hook up the Insert Image dialog

### ISSUE THREAD

**Issue by shanhuiyang** [nhn/tui.editor#675]
## Version
the latest, I am using the React Wrapper v1.0.1

## Test Environment
Chrome, OSX
## Current Behavior

The insert image dialog cannot be hooked.

## Expected Behavior
I would like to implement the dialog by myself so that the UI style can be consistent with my existing web app.

By the way, I would like to customize the style of the editor, such as border radius, focus highlight, etc. I found no options to customize that yet, is there such option?

**Comment by seonim-ryu** [comment]
@shanhuiyang 
If you just want to change the style, you can override the style of the selector in the css file below.
>
https://github.com/nhn/tui.editor/blob/51f336f7ad7b5f1a9de30c31cac8056bd02d1135/src/css/tui-editor.css#L501

**Comment by shanhuiyang** [comment]
Yes, thank you @seonim-ryu . But how could I change the layout of the dialog?

**Comment by seonim-ryu** [comment]
@shanhuiyang 
Unfortunately there is currently no way to change the layout.. :(
To do this, I need to consider how to provide a template for each popup in the editor.

**Comment by seonim-ryu** [comment]
@shanhuiyang 
Unfortunately there is currently no way to change the layout.. :(
To do this, I need to consider how to provide a template for each popup in the editor.

**Comment by shanhuiyang** [comment]
@seonim-ryu , well, only change the style usually cannot make the dialog be consistent with the websites' UX.

**Comment by shanhuiyang** [comment]
Also sometimes we would like to provide some image processing steps such as rotation and zooming before user upload the image. Then the embedded image select dialog does not provide hooks to do that.
